bluesky_queueserver/server/server.py

```diff
@@ -44,8 +44,12 @@ async def startup_event():
 
     # TODO: implement nicer exit with error reporting in case of failure
 
+    zmq_server_address = os.getenv("QSERVER_ZMQ_ADDRESS", None)
```

Comment:
I was thinking about having those common variables (such as the one here `QSERVER_ZMQ_ADDRESS`, and the others related to encryption) defined in a module somewhere, and they can be imported and reused from a single location, in a similar fashion as it was recently done in https://github.com/caproto/caproto/pull/727. That way we can avoid potential typos.

Reply:
Let's see how the project develops. I think that at this point there are very few variables and I think only one variable is used by two application. May be it's a good idea to define additional variables that could be used instead of command-line arguments. Then variable names could be consolidated.